src/pages/messaging/agent-chat-api/index.mdx

```diff
@@ -1583,6 +1583,16 @@ https://api.livechatinc.com/v3.4/agent/action/delete_event_properties \
 
 ### Tag Thread
 
+Tags a given thread. The following restrictions apply:
+
+- Active and inactive chats can be tagged.
+- One or more tags can be added to the specific **thread**.
+- To tag a chat, the **tag** needs to be created first. It can be done in the [LiveChat Application](https://accounts.livechat.com/) or via the [Configuration API](/management/configuration-api).
```

Comment:
```suggestion
- To tag a chat, the **tag** needs to be created first. It can be done in the [LiveChat Application](https://my.livechatinc.com/) or via the [Configuration API](/management/configuration-api).
```

Reply:
Or have I missed something and we redirect to the Accounts panel instead of the AA @ale-kacperczyk? 😅

Reply:
livechatINC ? someone needs "the talk" :trollface: 
anyway, I would avoid pushing the user towards AA when they are browsing API docs 😉

Reply:
Well, actually, https://my.livechatinc.com/ still has "inc". Joke's on you XD

Reply:
I didn't even consider that I need to verify it, and yet you are right, AA still uses "inc", good grief, "kajam się" XD

Reply:
but still, I don't think this is the place to mention AA; you check how to tag a thread with API, and to add a new tag we send you to this GUI app... meh

Reply:
Since we agreed to merge two points into one, this entire point can be deleted.

Reply:
Already deleted:)